app/scripts/modules/netflix/ci/ci.filter.model.ts

```diff
@@ -1,10 +0,0 @@
-import {module} from 'angular';
-
-export class CiFilterModel {
-  public searchFilter: string;
-}
-
-export const CI_FILTER_MODEL = 'spinnaker.netflix.ci.filter.model';
-
-module(CI_FILTER_MODEL, [])
```

Comment:
Oh, I hadn't noticed that it was a `static` field (so no `new`'ing necessary).  

I personally find the static field a bit awkward, and it's still a singleton.   The global redux store will address this, yes.